README.md

```diff
@@ -515,3 +515,19 @@ Check a complete example [here](https://github.com/feedhenry-staff/wfm-workorder
   }
 }
 ```
+
+##### Custom data collision handler setup
```

Comment:
Odd This is already on master .

https://github.com/feedhenry-raincatcher/raincatcher-sync#custom-data-collision-handler-setup

Does it need a rebase?